### DISCUSSION TITLE
2024-07-26 회의록

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#106] [회의록]
### 🔥 최종 프로젝트 일정 리마인드

<table>
  <tr>
    <th>최종 프로젝트 데드라인 테이블</th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/169b87c5-ae02-4f19-a787-0f970b424517" width="300" height="300"/></td>
  </tr>
</table>

- 생각보다 타이트하게 생각해야 할 듯.
  - 개발 시안 제작 완료 기한 => 7/30(화)까지
  - 31일(수)~8/2(금)까지 평일 3일 제작 후 주말 제외 후 바로 8/5(월)에 MVP 발표
    - MVP 발표를 위해 완료되어야 하는 작업
      - 일부 기능 구현을 제외한 모든 기능 구현 (UI의 완성은 아니더라도 핵심 기능은 전 인원 전부 해당)
      - 발표 : 오은
      - 발표 대본 작성 : 오은
      - readme 작성 : 김병준
      - 시연 영상 촬영 : 김민지
      - PPT 제작 : 곽현정
      - UT 관련 : 최유상
    - 위 발표를 위해서 8/3(토)~4(일) 주말을 희생하거나 평일 3일 중 하루는 개발 외 작업이 하루 종일 이루어질 것
    - 그렇다면 UI 시안 최종 완료본(모바일)을 최소 7/26(금)까지 받아본 후
    - 이번 주말 7/27(토)~28(일)까지는 기본적인 작업이 모두 끝나야
    - 다음 주 평일 2~3일 간 빌드 에러 또는 잔버그를 잡고 MVP 발표가 가능할 것으로 예상됨.
    - MVP 발표 시에는 말로 설명해도 되지만, 8/5(월, MVP 발표일) 다음 날인 8/6(화)부터는 User Test 시작 기간으로 User Test 내용도 미리 준비되어야 함.
      - 레퍼런스 체크는 슬랙의 다른 기수 내용 참고.
      - 우리 사이트에서 중점적으로 받아보고 싶은 피드백 내용 수렴 -> 구글 form에 작성
      - 상품 내용 등도 정해야 함.

### 🔥 디자이너 동반 회의

- 튜토리얼 바뀐 부분+순서 (페이지 구성 순서)
  -  상단 오른쪽 skip 버튼 추가 (기존 바로 시작하기 버튼 위치 이동)
  -  글 더미 데이터 공간
  -  이미지 더미 데이터 공간
  -  인디케이터
  -  다음버튼(퍼널패턴)

- 글쓰기 버튼 진입 flow 추가
  - 홈화면, 여정에서만 글쓰기 진입 가능한 플로팅 버튼 추가

### 🔥 개발 회의

- locationData에서 '해외' - '아시아' - '한국' 제외 요청

### 🔥 검색 페이지 공수 관련

- 현재 와이어프레임에 나와 있는 모바일, PC 버전은 검색 페이지의 작성 로직이 달라 공수가 클 것으로 보임 -> 모바일, PC 환경 둘 다 1페이지 내에서 검색과 
  - 모바일 검색은 위치, 날짜 선택은 별도의 컴포넌트로 이동해서 검색 데이터를 사용자에게 입력 받아야 하는데, 이 입력 받은 데이터를 호출한 페이지에 넘겨 받는 방식이 되어야 하는데, 검색 결과 페이지에서 상단에 계속 노출되어 있는 검색 버튼은 별도 페이지로 이동하지 않고 검색 결과 페이지 내에서 검색하게 되면 로직을 두 번 구현해야 하기 때문에 상당한 공수가 예상됨.
  - -> 구현은 가능하나, 상당한 공수가 들어갈 것으로 판단되어
  - -> 1페이지 내에서 아코디언으로 접는 방식으로 구현하는 것이 가장 현실적으로 보임. -> 이 방식이 UI/UX 관점에서 좋지 않다고 한다면 MVP 발표 이후 2주 간 도전 과제로 넘기는 것이 좋을 것 같음. (오은, 김병준, 곽현정 논의 결과)

## 🍀 공통 트러블 슈팅

- 검색 페이지 제작 공수가 높아 MVP 발표 이전 구현이 어려울 것 같다는 판단 (저녁 회의 내용 참고)

**Comment by slsl2** [comment]
# 저녁 회의 의견 (김민지)
### 와이어프레임 관련, 디자이너님께도 전달 필요.
1. 검색 결과 페이지 pc : 가로 스크롤을 모두 그리드로 바꾸고, 처음엔 두 줄만 띄워주고 '더보기'를 누르면 줄이 추가되는 거로 변경하면 좋을 것 같습니다.
2. trip 카드가 안에 내용은 거의 똑같은데 ui가 다른 버전으로 세 개가 있는데, 구분하지 않아도 된다면 통일하는 게 어떨까요? 안에 들어가는 내용이 trip 목록 페이지에서 더 풍부해진다면 구분하는 게 맞지만, 현재 비교해보니 비슷한 것 같아서 말씀 드려봅니다!
3. trip 카드의 버튼 **[참여하기]** / **[저장하기] [채팅하기]** & trip 상세 페이지의 버튼 **[찜하기] [참가하기]** 이름을 통일하면 좋을 것 같습니다.
- 왼쪽 버튼은 **찜하기**와 **저장하기** 두 단어 중 하나를 골라서 전부 통일 필요
- 오른쪽 버튼은 유저가 해당 trip의 참여자일 경우 [채팅하기]로, 참여자가 아닐 경우 [참여하기]로 통일하는 건 어떨까요!
4. trip에서 **[나가기]** 버튼은 상세 페이지에 버튼 영역 하나를 추가해서 넣으면 좋을 것 같다.. 고 회의 때 말씀 드렸었는데요... 지금 댓글 작성하면서 생각난 건데 **내 프로필 페이지에서만 나가기 버튼 누를 수 있도록 하자고** 전에 얘기 나눴던 것 같기도 합니다.

**Comment by eunohhh** [comment]
# 저녁 회의 의견 보충(오은)
- 데스크탑 모집중 여정 페이지는 페이지네이션보다 더보기가 좋을 것 같다는 의견입니다!

## 검색페이지관련
- 하나의 라우트 /search 로
- 키워드인풋 빼고 나머지 모두 아코디언으로
- 직관적으로 알아볼 수 있으려면 정보량을 줄일것...

**Comment by Captain-Kim** [comment]
## MVP 발표 기한 관련 (김병준)

- 시간 상 글쓰기 부분에서는 Rich-Text-Editor 구현 관련하여 MVP 발표 이후 과제로 넘기고 MVP 발표 이전에는 textarea로 임시 구현해야겠습니다.
- 글쓰기 부분에서 공수를 최대한 줄이고 팀원 분들 필수 기능을 함께 구현해서 완성시켜야 할 것 같습니다.
- 참여기업 튜터 피드백이 7/29(월)부터 예정되어 있어 기술 문서를 주말 간 작성 예정입니다.
